src/utils/hooks/useSplitTime.ts

```diff
@@ -7,7 +7,7 @@ const useSplitTime = (
   activeTimer: boolean,
   periodLength?: number,
 ) => {
-  const [timeLeft, setTimeLeft] = useState<number>(initialTime / 1000);
```

Comment:
With this change, the timer in `SaleStateWidget` is showing the wrong time:
![FireShot Capture 500 - Colony - localhost](https://user-images.githubusercontent.com/18473896/143618759-a045d575-050e-44d2-a0a4-4546f539be3c.png)

Correct time:

![FireShot Capture 501 - Colony - localhost](https://user-images.githubusercontent.com/18473896/143618794-58a40f58-7378-4387-899f-4078bcb3d030.png)

So I think we may be receiving different time units in another place, weird.